api/definitions.yaml

```diff
@@ -0,0 +1,260 @@
+ProblemJson:
```

Comment:
@gunzip  why don't

 - just C&P https://zalando.github.io/problem/schema.yaml so that we could just diff it?
 - still define HttpStatus for other purposes